runners/google-cloud-dataflow-java/worker/windmill/src/main/proto/windmill.proto

```diff
@@ -763,6 +763,8 @@ message StreamingGetWorkRequest {
     GetWorkRequest request = 1;
```

Comment:
// Initial message is GetWorkRequest with subsequent messages being extensions.

Reply:
done